app/assets/javascripts-es6/item-select.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(() => {
+  const items = document.getElementsByClassName("item");
+  for(let i=0; i<items.length; i++){
+    items[i].addEventListener("click", onItemSelect);
+  }
+});
+
+const onItemSelect = (e) => {
+  e.preventDefault();
+  document.getElementById("board-advice").innerHTML = "";
+  const itemNode = e.target.closest(".item");
+  const itemId = itemNode.getAttribute("data-item-id");
+  const section = itemNode.getAttribute("data-section-id");
+  const imgUrl = itemNode.getAttribute("data-large-img");
+  let imgNode = document.createElement("img");
```

Comment:
This should be `const`. `let` vs `const` has to do with _assignment_ not _mutibility_, and you should always use `const` if possible. I use this rule in all my projects: https://eslint.org/docs/rules/prefer-const

So, if you don't reassign to a variable later in a block, use `const`. Further, I tend to think of `let` as a "code smell". I try to find ways to write my code without it where possible. For example, if I'm reassigning a variable, chances are I should just use a new variable, since clearly the value has changed, and it probably deserves a new name.

Loops and conditionals are cases where it often can't be avoided.

Reply:
Okay this makes sense